qemu/tests/hugepage_mem_stress.py

```diff
@@ -1,9 +1,14 @@
 import logging
+import os
 
+from virttest import env_process
 from virttest import error_context
 from virttest import utils_misc
 from virttest import utils_test
+
 from virttest.utils_test import BackgroundTest
+from virttest.utils_test import utils_memory
+from avocado.utils import process
```

Comment:
Better to put avocado related importing to the front of VT's.

Reply:
correcting